packages/client/src/pages/Game.tsx

```diff
@@ -14,12 +14,52 @@ import { Board } from 'components/Board';
 import { GameScore } from 'components/GameScore';
 import { PlayerName } from 'components/Players/PlayerName';
 import { FiRotateCcw } from 'react-icons/fi';
+import { flip } from 'utils/animations';
 
 const SETTEBELLO = {
   value: 7,
   suit: Suit.Golds,
 };
 
+const animateCardOnTable = (activePlayerCard: string, options: Record<string, unknown> = {}) => {
+  const cardWrap = document.querySelector(`#card_${activePlayerCard}`)?.firstChild;
+  const dropElement = document.querySelector('#drop-container');
+  console.log(cardWrap, dropElement);
+  console.log('Emit: Card is going to the table');
+  function moveCard() {
+    dropElement?.removeChild(dropElement.firstChild!);
+    dropElement?.appendChild(cardWrap!);
+  }
+  flip([cardWrap], moveCard, options);
+  return cardWrap;
+};
+const animateCapture = (animatedCard: ChildNode, caputeredCards: string[], options: Record<string, unknown> = {}) => {
+  const cardWraps = caputeredCards
+    .map((cardKey) => document.querySelector(`#card_${cardKey}`)?.firstChild)
+    .concat([animatedCard]) as ChildNode[];
+  const dropElement = document.querySelector(`#player-deck`);
+  console.log(cardWraps, dropElement);
+  function moveCards() {
+    cardWraps.forEach((el) => {
+      // @ts-ignore
+      el.style.position = 'absolute';
+      // @ts-ignore
+      el.style.top = 0;
+      dropElement?.appendChild(el);
+    });
+  }
+  flip(cardWraps, moveCards, {
+    ...options,
+    onComplete: () => {
+      // @ts-ignore
+      options.onComplete();
+      cardWraps.forEach((el) => {
+        dropElement?.removeChild(el);
```

Comment:
What is happening here... `dropElement` is the player's deck. You first _flip_ (?), then move that appends cards to the player's deck, and then once all that is complete, you remove the cards from the deck...?